packages/core/src/utils/collect-updates/__tests__/collect-updates.spec.ts

```diff
@@ -74,7 +74,29 @@ describe("collectUpdates()", () => {
         name: "package-standalone",
       }),
     ]);
-    expect(hasTags).toHaveBeenLastCalledWith(execOpts);
+    expect(hasTags).toHaveBeenLastCalledWith(execOpts, '');
+    expect(describeRefSync).toHaveBeenLastCalledWith(execOpts, undefined, false);
+    expect(makeDiffPredicate).toHaveBeenLastCalledWith("v1.0.0", execOpts, undefined);
+  });
+
+  it("returns node with changes in independent mode", () => {
+    changedPackages.add("package-standalone");
+
+    const graph = buildGraph();
+    const pkgs = graph.rawPackageList;
+    const execOpts = { cwd: "/test", match: '*@*'};
```

Comment:
can you provide an example of a tag with this pattern match `*@*`? I mean what would a typical tag look like for this? Might be better to add a unit test with such example!?

Reply:
All tags tagged by lerna follow `@wundergraph/sdk@0.92.6`

Reply:
> Might be better to add a unit test with such example!?

Yes, let me check.